Gemfile.lock

```diff
@@ -322,6 +313,8 @@ GEM
     nokogiri (1.13.6)
       mini_portile2 (~> 2.8.0)
       racc (~> 1.4)
+    nokogiri (1.13.6-x86_64-linux)
```

Comment:
How have we ended up with 2 nokogiris?

Reply:
i assume the second one is specifying both version and platform. it needs this because nokogiri has binary components that need to be compiled and mixing up 32/64 bit systems would not work.